.github/workflows/asv-regular.yml

```diff
@@ -33,13 +33,14 @@ jobs:
           architecture: "x64"
       - name: Install dependencies
         run: |
+          sudo apt-get install -y libopencv-dev
           python -m pip install --upgrade pip
           pip install asv virtualenv
       - name: Add machine info for ASV
         run: asv machine --machine GH-Actions --os ubuntu-20.04 --arch x64 --cpu "2-core unknown" --ram 7GB
       - name: Run benchmarks for commits since v1.1
-        # On main the release of v1.1 is the 2.0dev tag
-        run: taskset -c 0 asv run --skip-existing-successful v2.0.dev..
+        # On main the release of v3.0 is the 4.0dev tag
```

Comment:
I'm confused: Doesn't the `v4.0dev tag` point to 3.1, not 3.0?

Reply:
Oh right I'm wrong. That's not how it works.

 I'll fix it